app/views/admin/applications/_application_details.html.erb

```diff
@@ -112,6 +112,24 @@
       row.action
     end
 
+    sl.row do |row|
+      row.key(text: 'Primary establishment')
+      row.value(text: application.primary_establishment.to_s)
+      row.action
+    end
+
+    sl.row do |row|
+      row.key(text: 'Number of pupils')
+      row.value(text: application.number_of_pupils)
```

Comment:
There is chance this could be `nil` can we ensure this always returns 0 `.to_i` maybe